app/policies/group_policy.rb

```diff
@@ -15,16 +15,18 @@ def destroy?
 
   def editable_attributes(all)
     return all if is_admin?
-
-    all - %i[members_count leaders_count neighbors_count rules_formatted
-             featured name pinned_post_id]
+    attrs = all
+    attrs -= %i[members_count leaders_count neighbors_count rules_formatted featured name]
+    attrs -= %i[pinned_post_id] unless record.owners.include? user
```

Comment:
I don't think this check is necessary, you can't `update?` the group unless you're a site admin or `has_group_permission? :community`